templates/index.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <title>Cityscape</title>
+        <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-1BmE4kWBq78iYhFldvKuhfTAU6auU8tT94WrHftjDbrCEXSU1oBoqyl2QvZ6jIW3" crossorigin="anonymous">
+        
+    </head>
+    <body>
+        {% block main %}
+        {% with errorMessage = get_flashed_messages() %}
+            {% if errorMessage %}
+            <p><strong>{{ errorMessage }}</strong></p>
+            {% endif %}
+        {% endwith %}
+        <div id="home-content" class="home-content">
```

Comment:
Any reason to include an `id` here but not on other divs?
Is the `id` needed? I don't see any references to it.

Reply:
Yeah there is no need for an `id` attribute here. It was originally in place in case I used JS for dynamic map so I can reference some `id` to that div, though I did not follow through with it and have stuck with static map imaging.